src/Promitor.Integrations.Sinks.OpenTelemetry/Configuration/OpenTelemetryCollectorSinkConfiguration.cs

```diff
@@ -1,7 +1,16 @@
-﻿namespace Promitor.Integrations.Sinks.OpenTelemetry.Configuration
+﻿using OpenTelemetry.Exporter;
+
+namespace Promitor.Integrations.Sinks.OpenTelemetry.Configuration
 {
     public class OpenTelemetryCollectorSinkConfiguration
     {
         public string CollectorUri { get; set; }
+        
+        public CollectorInfo Collector { get; set; }
+
+        public class CollectorInfo
+        {
+            public string CollectorProtocol { get; set; } = "grpc";
```

Comment:
Let's introduce Uri here as well to be consistent and deprecate the old one

Reply:
Added to new class but kept filename :)

Reply:
see other comment about class after I nuked the unit tests. Also, a change to the class structure or a nested class would maybe need runtime.yaml changes, which I think we should avoid?